src/read.c

```diff
@@ -1527,6 +1579,62 @@ void ReadFuncExpr1 (
 
 /****************************************************************************
 **
+*F  ReadFuncExprLong(<follow>) . . . . . read a multi-arg function expression
+**
+**  'ReadFuncExprLong' reads  an abbreviated  function literal expression.  In
```

Comment:
Again a comment line is a bit too long. Just remove some of the double spaces (I think trying to justify format our comments is a silly waste of time anyway).

Of course you may argue that limiting line lengths is silly, too ;-). But then I'd get rid of the `/*****...`, too, as it looks weird to have that and then go wider right after it...